libs/common/src/vault/models/domain/cipher.ts

```diff
@@ -123,9 +120,13 @@ export class Cipher extends Domain implements Decryptable<CipherView> {
     }
   }
 
-  async decrypt(cipherService: CipherService, encKey?: SymmetricCryptoKey): Promise<CipherView> {
+  async decrypt(encKey?: SymmetricCryptoKey): Promise<CipherView> {
     const model = new CipherView(this);
-    encKey = encKey ?? (await cipherService.getCipherKey(this.organizationId));
+
+    if (encKey == null) {
+      const cryptoService = Utils.getContainerService().getCryptoService();
+      encKey = await cryptoService.getKeyForCipherKeyEncryption(this.organizationId);
```

Comment:
```suggestion
      encKey = await (this.organizationId != null ? cryptoService.getOrgKey(this.organizationId) : cryptoService.getKeyForUserEncryption();
```

As said in crypto service, I feel this logic is better suited here. wrote this without IDE help, btw, please double check 😏